assets/panel/debugger.properties

```diff
@@ -14,6 +14,11 @@
 # that collapses the left and right panes in the debugger UI.
 collapsePanes=Collapse panes
 
+# LOCALIZATION NOTE (copyToClipboard): This is the text that appears in the
+# context menu to copy the complete source of file open.
+copyToClipboard=Copy to clipboard
```

Comment:
Please use copyToClipboard.label for this one (and update the reference in the commit accordingly)

nit on comment: "of file open" -> "of the open file"